samples/hosting/docker/sample.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Hosting endpoints in Docker Linux containers
+summary: Hosting multiple endpoints in several Docker Linux containers managed by Docker Compose
+reviewed: 2017-09-25
+component: Core
+tags:
+- Hosting
+related:
+- nservicebus/hosting
+---
+
+This sample demonstrates how to use Docker Linux containers to host NServiceBus endpoints communicating over the RabbitMQ transport.
+
+## Prerequisites
+
+This sample requires that the following tools are installed:
+
+  - [.NET Core 2.0 SDK](https://www.microsoft.com/net/download/core)
+  - [Docker Community Edition](https://www.docker.com/community-edition) or higher
+
+NOTE: Container that runs the RabbitMQ broker is a Linux container and [Docker for Windows also needs to be configured to use Linux containers](https://docs.docker.com/docker-for-windows/#switch-between-windows-and-linux-containers).
+
+## Running the sample
+
+The sample requires .NET Core CLI tools and Docker CLI tools to build and run the code.
+
+### Building and publishing binaries
+
+First step is to build the binaries using the .NET Core command line tools:
+
+```
```

Comment:
Can you [pick a language](https://github.com/isagalaev/highlight.js/tree/master/src/languages) for all these command-line snippets? `dos`? `bash`? Otherwise it's possible highlight.js might pick something crazy and highlight odd things.

Also prefixing command prompts with a `>` (like [NuGet uses](https://www.nuget.org/packages/nservicebus) on the .NET CLI tab) might be helpful.

Reply:
@DavidBoike we have HL.js configured to have no default language https://github.com/Particular/DocsEngine/commit/82c7a97efa18cdb72fb0ce048bc4bfe0ced2c5f3